### PR TITLE
[AGW] OVS: build script: 2.8.9 package

### LINKED ISSUE #2538: Update OVS to handle vlan with GTP tunnels

### PR DESCRIPTION
**Pull request by pshelar**
## Summary

This patch updates OVS version to latest 2.8 release and includes
vlan fix required for multi APN datapath

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

## Test Plan
`make test`

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Comment by tmdzk** [comment]
Okay let me add some stuff to update that. 
but looks good

File: lte/gateway/release/build-ovs.sh [hunk 1]
```diff
@@ -40,14 +40,15 @@
 
 set -e
 WORK_DIR=~/build-ovs
-OVS_VERSION="v2.8.1"
-OVS_VERSION_SHORT="2.8.1"
+OVS_VERSION="v2.8.9"
+OVS_VERSION_SHORT="2.8.9"
 MAGMA_ROOT="../../../"
-GTP_PATCH_PATH="${MAGMA_ROOT}/third_party/gtp_ovs"
+GTP_PATCH_PATH="${MAGMA_ROOT}third_party/gtp_ovs/kenrel-4.9"
```
Comment (xjtian) [inline lte/gateway/release/build-ovs.sh]: kenrel -> kernel and rename the directories appropriately

leave the / after MAGMA_ROOT just for readability

File: lte/gateway/release/build-ovs.sh [hunk 2]
```diff
@@ -81,9 +82,11 @@ cd ${WORK_DIR}
 git clone https://github.com/openvswitch/ovs.git
 cd ovs
 git checkout ${OVS_VERSION}
-cp ${GTP_PATCH_PATH}/ovs/${OVS_VERSION_SHORT}/*.patch "${WORK_DIR}/ovs"
+cp $PATCH_ROOT/*.patch $WORK_DIR/ovs
 cp -r "${FLOWBASED_PATH}" "${WORK_DIR}/ovs/flow-based-gtp-linux-v4.9"
-git apply ${PATCHES}
+git am ${PATCHES}
+# vlan fix
+git cherry-pick 3cf2b424bb
```
Comment (xjtian) [inline lte/gateway/release/build-ovs.sh]: declare the hash as a var up top

File: lte/gateway/release/build-ovs.sh [hunk 3]
```diff
@@ -99,3 +102,6 @@ ksrc="/lib/modules/$kvers/build"
 sudo make -f debian/rules.modules KSRC="$ksrc" KVERS="$kvers" binary-modules
 
 cp /usr/src/*.deb ${WORK_DIR}
+
+# OVS packages:
+# sudo dpkg -i openvswitch-datapath-module-4.9.0-9-amd64_2.8.9-1_amd64.deb openvswitch-switch_2.8.9-1_amd64.deb openvswitch-test_2.8.9-1_all.deb python-openvswitch_2.8.9-1_all.deb  libopenvswitch_2.8.9-1_amd64.deb openvswitch-common_2.8.9-1_amd64.deb
```
Comment (xjtian) [inline lte/gateway/release/build-ovs.sh]: ?

**Review by xjtian — CHANGES_REQUESTED** [review]

File: lte/gateway/release/build-ovs.sh (hunk 3 above)
Comment (pshelar) [reply lte/gateway/release/build-ovs.sh]: I moved it to commit msg for list of packages to install.

**Review by xjtian — APPROVED** [review]
Try to test this by building the package and installing it on the magma-prod VM. Does this need to land in 1.2?

**Comment by pshelar** [comment]
yes, I have tested it on AGW, it builds OVS packages as expected.

**Pull request merged**